app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,41 @@
+// Space out content a bit
+body {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+// Custom page header
+.header {
+  padding-left: 15px;
+  padding-right: 15px;
+  margin-bottom: 15px;
+  border-bottom: 1px solid #e5e5e5;

```

Comment:
Color literals like `#e5e5e5` should only be used in variable declarations; they should be referred to via variable everywhere else.